frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/storage-tab-validation.ts

```diff
@@ -60,7 +58,7 @@ export const validateStorages = (options: UpdateOptions) => {
           {
             usedDiskNames,
             usedPVCNames,
-            allowedBusses,
+            templateValidations,
```

Comment:
ok let's pass templateValidations and hope we don't need that abstracted more in the future. I guess we can use those even in vm details disk row